Dockerfile

```diff
@@ -16,5 +16,8 @@ RUN npm install --production
 # Bundle app source
 COPY src/ /usr/src/app/src/
 
+# Bundle init scripts
+COPY scripts/ /usr/src/app/scripts/
```

Comment:
would it just be better to put under /src?

Reply:
I'm ok with that. Especially since people won't need to do anything with it anymore.